composer.json

```diff
@@ -40,16 +40,14 @@
         "mezzio/mezzio-helpers": "^5.0",
         "mezzio/mezzio-router": "^3.0",
         "mezzio/mezzio-template": "^2.0",
-        "psr/container": "^1.0",
+        "psr/container": "^1.0 || ^2.0",
```

Comment:
`psr/container                                  1.1.2     Common Container...` is installed in latest builds: let's revert this, as it's obviously not tested.